### PR TITLE
fix: Fixed Semantic changes in Multiple HTML Files

### PR DESCRIPTION
**Pull request by NarendraKoya999**
### 🛠️ Fixes Issue 
- **This PR is linked to Old PR #1253 . Because of, Some conflict Issues Placed in My Local Project Directory**

### 👨‍💻 Changes proposed

- **Modified and Added Semantic Tags `<header>, <main>, <section>` in `contributors.html, about.html, blogs.html, cbse_notes.html, index.html, ncert.html, resource.html, reviewsection.html` Files.**
- **In Some Files Remove `<center></center> ` Tag**
- **`For Reviewers, Observe My Changes in Files Changed File`**

### ✔️ Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers

- **This PR is linked to Old PR #1253 . Because of, Some conflict Issues Placed in My Local Project Directory of Old PR. Atleast this PR raised 2 months ago**
- **Please taken my Requests into Consideration. Take review on this. If applicable, Merge this ?**

### CODE REVIEW

**Comment by rohanbabbar04** [comment]
Issue no.??

**Comment by NarendraKoya999** [comment]
> 

Hi @rohanbabbar04 . This is Direct PR and It has no Issue No. At the starting of the GSSOC Program., I don't know  First Create Issue and Then Create a New PR

**Comment by rohanbabbar04** [comment]
Then can you explain it a little bit what change does it bring...

**Comment by NarendraKoya999** [comment]
> Then can you explain it a little bit what change does it bring...



> Then can you explain it a little bit what change does it bring...

### This is my What Changes Done Before and After Modifications
I have modified HTML5 Layout Errors in contributors.html File
BEFORE Modification:
`<nav</nav> , 3 <div></div> sections in the <body></body>` section in `contributors.html`
Live Code :
```
<body>
   <nav></nav>
   <div></div>
  <div></div>
  <div><footer></footer></div>
</body>
```
AFTER Modification:
`<header></header>,<nav></nav>,<main></main>, 2 <section></section>` tags added in contribution.html
```
<body>
  <header>
       <nav></nav>
  </header>
  <main>
     <section>
         <div></div>
    </section>
    <section>
         <div></div>
    </section>
  </main>
  <footer></footer>  // Remove <div></div>tag around the <footer></footer> section.
</body>
```

**Comment by rohanbabbar04** [comment]
One question @NarendraKoya999 why only in `contributors.html` and not other files ?

**Comment by NarendraKoya999** [comment]
> One question @NarendraKoya999 why only in `contributors.html` and not other files ?

I have Individually raised a PR's for remaining Files Also. Kindly check @rohanbabbar04

**Comment by rohanbabbar04** [comment]
What is the point of making seperate PR's for more or less the same issue, add all of them in this PR itself..

**Comment by NarendraKoya999** [comment]
> What is the point of making seperate PR's for more or less the same issue, add all of them in this PR itself..

At the Starting of the Program. I don't have Much Knowledge on How to Create PR's on Particular same relevant Issues

**Comment by rohanbabbar04** [comment]
Don't worry 
Just add the semantic changes of all the files in this PR only...

**Comment by NarendraKoya999** [comment]
> Don't worry Just add the semantic changes of all the files in this PR only...
> 
> And close other PRs(as you created for each file)

Ok. I can work on that. Removed unnecessary PR's and Changes adding in this PR and It's Branch

**Comment by rohanbabbar04** [comment]
Yup, Sound good to me
One done just request a review...

**Comment by rohanbabbar04** [comment]
Thanks I will review it soon....

**Comment by NarendraKoya999** [comment]
> Thanks I will review it soon....

Hi @rohanbabbar04 . Any Update on PR?. Please Review on this?

**Comment by rohanbabbar04** [comment]
@NarendraKoya999 I'll review it over the weekend

File: resource.html [hunk 1]
```diff
@@ -859,7 +859,7 @@ <h3 class="hover-underline-animation">Data Structure and Algorithm Tutorials</h3
             <br><br>
         </section>
 
-        <center>
+
```
Comment (rohanbabbar04) [inline resource.html:862]: Revert this, this should be centered aligned...

Comment (NarendraKoya999) [reply resource.html]: I reverted Back Center Tag in resource.html File

Comment (rohanbabbar04) [reply resource.html]: Thanks looks good...

**Review by rohanbabbar04 — APPROVED** [review]

**Pull request merged**